src/pool/pool_scalable.c

```diff
@@ -167,19 +173,88 @@ static void tbb_raw_free_wrapper(intptr_t pool_id, void *ptr, size_t bytes) {
     }
 }
 
+umf_result_t
+umfScalablePoolParamsCreate(umf_scalable_pool_params_handle_t *params) {
+    if (!params) {
+        LOG_ERR("params is NULL");
```

Comment:
nit: This message could be more expressive, like the memory allocation error log below, `invalid scalable pool params handle` or something.

Reply:
Done